server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -239,6 +239,38 @@ func (s *BuildBuddyServer) CreateGroup(ctx context.Context, req *grpb.CreateGrou
 	}, nil
 }
 
+func (s *BuildBuddyServer) UpdateGroup(ctx context.Context, req *grpb.UpdateGroupRequest) (*grpb.UpdateGroupResponse, error) {
+	auth := s.env.GetAuthenticator()
+	userDB := s.env.GetUserDB()
+	if auth == nil || userDB == nil {
+		return nil, status.UnimplementedError("Not Implemented")
+	}
+	if err := s.authorizeGroupAccess(ctx, req.GetId()); err != nil {
+		return nil, err
+	}
+	group := &tables.Group{
+		GroupID: req.GetId(),
+	}
+	if err := userDB.FillGroup(ctx, group); err != nil {
+		return nil, err
+	}
+	group.Name = req.GetName()
```

Comment:
can this live above when you define the group?

group := &tables.Group{
                Name: req.GetName(),
		GroupID: req.GetId(),
	}

Reply:
I had the same thought, but realized `FillGroup` would clobber the name. Yet another reason to get rid of `FillGroup`

Reply:
yuck! can't wait to be rid of that thing.